### PR TITLE
Sync some `uses_from_macos` upstream from Homebrew/linuxbrew-core

### PR DESCRIPTION
**Pull request by issyl0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Over in Homebrew/linuxbrew-core, we fix builds frequently using `uses_from_macos` for macOS built-in dependencies like `libxml2`, `libxslt` and `curl`. Sometimes, we want to get changes in quickly so we don't raise PRs upstream and go through the merge cycle. I noticed some of these earlier and thought we should sync up so that we're as consistent as possible - given that's what the `uses_from_macos` DSL is designed for.

These should all be a no-op on macOS and therefore not require bottles. So this should be good to merge via the GitHub UI.

### CODE REVIEW

File: Formula/latexml.rb [hunk 1]
```diff
@@ -13,6 +13,10 @@ class Latexml < Formula
     sha256 "b911ac9897012edcc7c32d96785e4ca3830ce8cbddff78da0942263c7fb0d0bb" => :sierra
   end
 
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+  uses_from_macos "perl"
+
```
Comment (Bo98) [inline Formula/latexml.rb]: `brew style` is picking up on trailing whitespace on this line

Comment (issyl0) [reply Formula/latexml.rb]: Thanks. And it turns out some `—online` audits are failing. 😳

Comment (issyl0) [reply Formula/latexml.rb]: Fixed style and audits that I could reproduce the failures for.

**Comment by Bo98** [comment]
sratoolkit test seems a little dodgy. It worked the first time round and is clearly not caused by this pull request so this should be good to merge.

**Pull request merged**